#
# Re-enable musl compilation in Slint that was removed via PR at
# https://github.com/slint-ui/slint/pull/8450
#
--- a/api/rs/slint/lib.rs	2025-08-19 21:34:04.056049759 -0400
+++ b/api/rs/slint/lib.rs	2025-08-19 21:34:30.025092220 -0400
@@ -201,9 +201,6 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 #![allow(clippy::needless_doctest_main)] // We document how to write a main function
 
-#[cfg(target_env = "musl")]
-compile_error!("Compiling with MUSL is not supported by this crate.");
-
 extern crate alloc;
 
 #[cfg(not(feature = "compat-1-2"))]
